### ISSUE TITLE
Implement decision_path in scikit.js

### ISSUE THREAD

**Issue by kenluck2001** [javascriptdata/scikit.js#262]
**Is your feature request related to a problem? Please describe.**
This is to return the path that shows how the decision tree makes decision on a label from the root to the leaf. My guess is that changes are needed [here](https://github.com/javascriptdata/scikit.js/blob/main/src/tree/DecisionTree.ts). Kindly provide a description of how it can be implemented, even if it is outside your project map.

**Describe the solution you'd like**

The Python implementation of [scikit-learn](https://github.com/scikit-learn/scikit-learn) has a  `def decision_path(self, X, check_input=True)` in the [source code](https://github.com/scikit-learn/scikit-learn/blob/da08f3d99/sklearn/tree/_classes.py#L585)

Provide suggestions on how it can be implemented in [scikit.js](https://github.com/javascriptdata/scikit.js).

**Describe alternatives you've considered**
A clear and concise description of any alternative solutions or features you've considered. This feature exist in [scikit-learn](https://github.com/scikit-learn/scikit-learn).
